### PR TITLE
Add test case with estimated initial conditions specified via the con…

### LINKED ISSUE #62: Add test case with estimated initial conditions specified via the conditions table.
Related to https://github.com/AMICI-dev/AMICI/issues/2455

Add a test with something like

conditionId | B | A
-- | -- | --
c0 | initialB | initialA

with estimated initialA or initalB

### PR DESCRIPTION
**Pull request by dweindl**
…ditions table

Closes #62

### CODE REVIEW

File: petabtests/cases/v1.0.0/sbml/0019/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# PEtab test case 0019
+
+## Objective
+
+This case tests handling of initial concentrations that are specified
+in the conditions table. For species `A`, the initial concentration is
+estimated for species `B` the initial concentration is specified in the
+parameters table.
```
Comment (dilpath) [inline petabtests/cases/v1.0.0/sbml/0019/README.md:8]:
```suggestion
This case tests handling of initial concentrations that are specified
in the conditions table. For species `A`, the initial concentration is
estimated. For species `B`, the initial concentration is specified in the
parameters table.
```

File: petabtests/cases/v1.0.0/sbml/0019/conversion_modified.xml
Comment (dilpath) [inline petabtests/cases/v1.0.0/sbml/0019/conversion_modified.xml:1]: Is this meant to match `_model.xml` exactly?

File: petabtests/cases/v2.0.0/sbml/0020/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# PEtab test case 0020
+
+## Objective
+
+This case tests handling of initial concentrations that are specified
+in the conditions table. For species `A`, the initial concentration is
+estimated for species `B` the initial concentration is specified in the
+parameters table.
```
Comment (dilpath) [inline petabtests/cases/v2.0.0/sbml/0020/README.md:8]:
```suggestion
This case tests handling of initial concentrations that are specified
in the conditions table. For species `A`, the initial concentration is
estimated. For species `B`, the initial concentration is specified in the
parameters table.
```

File: petabtests/cases/v2.0.0/sbml/0020/0020.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+from petab.C import *
+
+from petabtests import PetabTestCase, analytical_a
+
+DESCRIPTION = cleandoc("""
+## Objective
+
+This case tests handling of initial concentrations that are specified
+in the conditions table. For species `A`, the initial concentration is
+estimated for species `B` the initial concentration is specified in the
```
Comment (dilpath) [inline petabtests/cases/v2.0.0/sbml/0020/0020.py:13]:
```suggestion
estimated. For species `B`, the initial concentration is specified in the
```

**Review by dilpath — APPROVED** [review]

File: petabtests/cases/v1.0.0/sbml/0019/conversion_modified.xml
Comment (dweindl) [reply petabtests/cases/v1.0.0/sbml/0019/conversion_modified.xml:1]: Currently, yes. Likely to be changed in the future. Then each test will probably specify its own model in antimony.

**Pull request merged**